src/test/java/org/cactoos/list/ImmutableTest.java

```diff
@@ -373,41 +392,87 @@ public void subList() {
     }
 
     @Test
-    public void testToString() {
+    public void immutableSubList() {
         new Assertion<>(
-            "toString() must be equals to original",
-            new Immutable<>(
+            "subList() must be immutable",
```

Comment:
@iakunin please, change is to `"subList() result must be immutable"`.

Reply:
@fabriciofx fixed